docs/reference/query-languages/esql/esql-commands.md

```diff
@@ -666,10 +666,6 @@ FROM employees
 ## `LOOKUP JOIN` [esql-lookup-join]
 
 ::::{warning}
-```{applies_to}
-stack: preview 9.0, coming 9.1
-serverless: preview
```

Comment:
I agree that this maybe shouldn't be in the tech preview warning, but I don't know if these tags should be removed altogether. Maybe @leemthompo can suggest a better position?